src/Peering/Peering/help/Remove-AzPeeringServicePrefix.md

```diff
@@ -0,0 +1,247 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Peering.dll-Help.xml
+Module Name: Az.Peering
+online version: https://docs.microsoft.com/en-us/powershell/module/az.peering/remove-azpeeringserviceprefix
+schema: 2.0.0
+---
+
+# Remove-AzPeeringServicePrefix
+
+## SYNOPSIS
+Removes a new peering service prefix
+
+## SYNTAX
+
+### ByName (Default)
+```
+Remove-AzPeeringServicePrefix [-ResourceGroupName] <String> [-Name] <String> [-PeeringServiceName] <String>
+ [-Force] [-AsJob] [-PassThru] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### Default
+```
+Remove-AzPeeringServicePrefix -InputObject <PSPeeringServicePrefix> [-Force] [-AsJob] [-PassThru]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ByResourceId
+```
+Remove-AzPeeringServicePrefix [-ResourceId] <String> [-Force] [-AsJob] [-PassThru]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Removes a peering service prefix from a peering service.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-AzPeeringService -ResourceGroupName $rgName -Name $peeringServiceName | Remove-AzPeeringServicePrefix -Name $prefixName
+
+True
+```
+
+Remove a prefix from a peering service object
+
+### Example 2
+```powershell
+PS C:\> Remove-AzPeeringServicePrefix -ResourceId $peeringServiceResourceId -Name $prefixName 
+
+True
```

Comment:
@rileymckenna this would only return `true` if the `-PassThru` parameter was provided, otherwise there's no output